app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -288,10 +288,21 @@ export const DirectoryLiveTable: React.FC<DirectoryTableProps> = (props) => (
                 Looker
               </LinkExternal>
             )}
+            {experiment.monitoringDashboardUrl &&
+              experiment.resultsReady &&
+              experiment.rolloutMonitoringDashboardUrl && <br />}
+            {experiment.rolloutMonitoringDashboardUrl && (
```

Comment:
Oh I just realized, we have to guard this somewhere that it is `isRollout` right?  We can either do it here in the UI or we can do it above in the model method.  We had this same pattern all the time in legacy, do you check in the UI whether to show something or do it in the model method.  Doing it in the model method means you don't hagve to worry about it later when you request and use it in the UI in arbitrary places, so it's maybe a little more reusable?

Reply:
I don't think we need to guard here. This is following the same pattern as the normal monitoring dashboard, as in it's either there or it's not. It might save us a line or two, but I don't think it's super necessary

Reply:
No I mean, if we don't guard the model method or the UI then there will be an opmon link for experiments that goes nowhere?

Reply:
oh, duh. adding that